content/tutorials/rust/mysql-cdc.md

```diff
@@ -0,0 +1,199 @@
+"Consumer", since it "consumes" messages from Fluvio.
+
+To get the most out of this tutorial, be sure to clone our [CDC demo repo] and follow
+along with the setup and commands.
+
+[CDC demo repo]: https://github.com/infinyon/fluvio-demo-apps-rust
+
+## Requirements
+
+This tutorial requires you to have completed the following prerequisites:
+
+- Install [Docker]
+- Install the [mysql client]
+- Complete the [Fluvio getting started guide]
+  - Note: You may use [Fluvio Cloud] or install [Fluvio locally], but you don't need both
+
+[Docker]: https://docs.docker.com/engine/install/
+[Fluvio getting started guide]: https://fluvio.io/docs/getting-started
+[Fluvio Cloud]: https://fluvio.io/docs/getting-started/fluvio-cloud/
+[Fluvio locally]: https://fluvio.io/docs/getting-started/fluvio-local/
+
+## Docker MySQL deployment
+
+This example requires two instances of MySQL to be up and running: One to be a
+leader that produces events, and one to be a follower that consumes events.
+We've provided docker images to help you set up these MySQL instances with
+the right configurations. Run the following commands to start your MySQL containers:
+
+```bash
+cdc-mysql$ cd docker/
+cdc-mysql/docker$ ./install.sh -n mysql-producer -d ~/mysql-cdc/mysql-producer -p 3080
+cdc-mysql/docker$ ./install.sh -n mysql-consumer -d ~/mysql-cdc/mysql-consumer -p 3090
+```
+
+At this point, you should be able to see both docker containers running
+
+```bash
+$ docker ps
+CONTAINER ID   IMAGE      COMMAND                  CREATED       STATUS       PORTS                               NAMES
+17c60cbbfc09   mysql-80   "docker-entrypoint.s…"   3 hours ago   Up 3 hours   33060/tcp, 0.0.0.0:3090->3306/tcp   mysql-consumer
+f5ec27a58476   mysql-80   "docker-entrypoint.s…"   3 hours ago   Up 3 hours   33060/tcp, 0.0.0.0:3080->3306/tcp   mysql-producer
+```
+
+## Creating a Fluvio Topic
+
+In Fluvio, every message is sent to a Topic. A topic is a sort of category for events that
+are related. For this example, we're going to create a topic that will receive all of our
+MySQL events. Run the following command to create the topic:
+
+```bash
+$ fluvio topic create rust-mysql-cdc
+```
+
+## Start the Producer and Consumer
+
+Now we'll launch the CDC Producer, which will watch for any SQL commands that are executed
+in the leader MySQL instance and produce corresponding Fluvio events.
+
+```bash
+cdc-mysql$ cargo run --bin cdc-producer -- ./producer_profile.toml
+```
+
+In another terminal window, we'll launch the CDC Consumer, which listens for new Fluvio
+events and replicates them in the follower MySQL instance.
+
+```bash
+cdc-mysql$ cargo run --bin cdc-consumer -- ./consumer_profile.toml
+```
+
+## Connect to Mysql
+
+Now you're ready to start interacting with your databases. We'll open up
+the `mysql` command line and start running queries on the leader database, and we'll
+open a separate window connected to the follower database to see that all the changes
```

Comment:
window -> terminal (you may want to restructure this sentence to emphasize on 2 different terminals).